x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/add_prebuilt_rules_table/add_prebuilt_rules_table_context.tsx

```diff
@@ -68,17 +68,13 @@ export interface AddPrebuiltRulesTableState {
   /**
    * Rule rows selected in EUI InMemory Table
    */
-  selectedRules: RuleInstallationInfoForReview[];
-  /**
-   * Rule that is currently displayed in the flyout or null if flyout is closed
-   */
-  flyoutRule: RuleInstallationInfoForReview | null;
+  selectedRules: RuleResponse[];
   /**
    * Is true when the install button in the flyout is disabled
    * (e.g. when the rule is already being installed or when the table is being refetched)
    *
    **/
-  isFlyoutInstallButtonDisabled: boolean;
+  canPreviewedRuleBeInstalled: boolean;
```

Comment:
Same here: This state is not used outside the flyout, so we don't need to expose it. Let's keep it internal.